src/components/cards/AirdropCard.svelte

```diff
@@ -183,22 +184,22 @@
 							minute: '2-digit'
 						})}~
 					</div>
-					<div class="flex items-center justify-center font-bold">Airdrop end</div>
-					<div class="flex items-center justify-center">
-						{new Date(thisDrop.claim.dates.claimDateClose).toLocaleString(undefined, {
-							day: '2-digit',
-							month: '2-digit',
-							year: 'numeric',
-							hour: '2-digit',
-							minute: '2-digit'
-						})}~
-					</div>
-					<div class="flex items-center justify-center font-bold">Duration</div>
-
-					<div class="flex items-center justify-center">
-						{thisDrop.claim.dates.claimDuration}~
-					</div>
 				{/if}
+				<div class="flex items-center justify-center font-bold">Airdrop end</div>
+				<div class="flex items-center justify-center">
+					{thisDrop.claim.dates.claimDateClose ? new Date(thisDrop.claim.dates.claimDateClose).toLocaleString(undefined, {
+						day: '2-digit',
+						month: '2-digit',
+						year: 'numeric',
+						hour: '2-digit',
+						minute: '2-digit'
+					}) : "unknown"}~
```

Comment:
kind of went in two directions here: hard coded `unknown` as a fallback for `claimDateClose`, but left `claimDuration` as-is and put `unknown` as the value in the config

lmk if you prefer one or the other or a secret third thing

Reply:
I made this more explict in template logic. Bit more code, less to digest looking at.